web/src/features/moreCast2/components/ForecastDataGrid.tsx

```diff
@@ -15,21 +15,50 @@ import { LinearProgress } from '@mui/material'
 import { DataGridColumns } from 'features/moreCast2/components/DataGridColumns'
 import ApplyToColumnMenu from 'features/moreCast2/components/ApplyToColumnMenu'
 import { ModelChoice, ModelType } from 'api/moreCast2API'
+import {
+  MORECAST_MODEL_COLORS,
+  MORECAST_WEATHER_PARAM_COLORS,
+  MoreCastModelColors,
+  MoreCastParamColors
+} from 'app/theme'
 
 const PREFIX = 'ForecastDataGrid'
 
 const classes = {
   root: `${PREFIX}-root`
 }
 
-const Root = styled('div')(() => ({
-  [`&.${classes.root}`]: {
-    display: 'flex',
-    flexGrow: 1,
-    flexDirection: 'column',
-    height: '1px'
+const Root = styled('div')(() => {
+  const styles: Record<string, React.CSSProperties> = {
+    [`&.${classes.root}`]: {
+      display: 'flex',
+      flexGrow: 1,
+      flexDirection: 'column',
+      height: '1px'
+    }
   }
-}))
+
+  Object.keys(MORECAST_WEATHER_PARAM_COLORS).forEach(key => {
+    styles[`& .${key}`] = {
+      backgroundColor: MORECAST_WEATHER_PARAM_COLORS[key as keyof MoreCastParamColors].active
+    }
+  })
+
+  Object.keys(MORECAST_MODEL_COLORS).forEach(key => {
+    styles[`& .${key}`] = {
+      backgroundColor: MORECAST_MODEL_COLORS[key as keyof MoreCastModelColors].bg
+    }
+    styles[`& .${key}-header`] = {
+      backgroundColor: MORECAST_MODEL_COLORS[key as keyof MoreCastModelColors].bg,
+      borderBottom: 'solid',
+      borderRight: 'solid',
+      // Ugly override, tried to avoid, but MUI overwrites border with it's own otherwise
+      borderColor: `${MORECAST_MODEL_COLORS[key as keyof MoreCastModelColors].border} !important`
+    }
+  })
+
+  return styles
```

Comment:
Nice! I didn't realize you could do this!

Reply:
Me neither, I learned this from @brettedw when I took over the PR last week

Reply:
Can't take all the credit, found an example of someone doing something similar 🙂